### PR TITLE
feat: 'fork me' ribbon

### PR DESCRIPTION
**Pull request by glebez**
Hi,

this PR is related to #195 issue and builds up on PR #647, that didn't show any activity for a while.

I've rebased wkwiatek's branch and added some tweaks based on feedback in the above PR.

Hope I got the requirements right and let me know, please, if any other changes are needed.

### CODE REVIEW

File: src/rsg-components/Ribbon/index.js [hunk 1]
```diff
@@ -0,0 +1 @@
+export { default } from './Ribbon.js';
```
Comment (sapegin) [inline src/rsg-components/Ribbon/index.js]: Here should be a full path with `rsg-components` too.

File: src/rsg-components/StyleGuide/StyleGuideRenderer.js [hunk 2]
```diff
@@ -4,6 +4,7 @@ import Logo from 'rsg-components/Logo';
 import Markdown from 'rsg-components/Markdown';
 import Styled from 'rsg-components/Styled';
 import cx from 'classnames';
+import Ribbon from '../Ribbon';
```
Comment (sapegin) [inline src/rsg-components/StyleGuide/StyleGuideRenderer.js]: And here.

File: src/rsg-components/Ribbon/RibbonRenderer.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+			</a>
+		</div>
+	);
+}
+
+RibbonRenderer.propTypes = {
+	classes: PropTypes.object.isRequired,
+	url: PropTypes.string.isRequired,
+	text: PropTypes.string,
+	color: PropTypes.string,
```
Comment (sapegin) [inline src/rsg-components/Ribbon/RibbonRenderer.js]: You don't use `color` and `background` props.

File: src/rsg-components/Ribbon/RibbonRenderer.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+		transform: 'rotate(45deg)',
+		cursor: 'pointer',
+	},
+});
+
+export function RibbonRenderer({ classes, url, text }) {
+	return (
+		<div className={classes.root}>
+			<a href={url} className={classes.link}>
+				{text || 'Fork me on GitHub'}
```
Comment (sapegin) [inline src/rsg-components/Ribbon/RibbonRenderer.js]: This could be a `defaultProps`.

File: src/rsg-components/Ribbon/RibbonRenderer.js [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+		zIndex: 999,
+	},
+	link: {
+		fontFamily: fontFamily.base,
+		position: 'relative',
+		right: -37,
+		top: -22,
+		display: 'block',
+		width: 190,
+		padding: '5px 15px',
```
Comment (sapegin) [inline src/rsg-components/Ribbon/RibbonRenderer.js]: Try to use `space` variables from theme here. (I'm not sure it's a good idea, but worth trying ;-)

**Comment by glebez** [comment]
Aaand the fixes are there.

File: src/rsg-components/Ribbon/RibbonRenderer.js [hunk 6]
```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Styled from 'rsg-components/Styled';
+
+export const styles = ({ color, space, fontSize, fontFamily }) => ({
+	root: {
+		position: 'fixed',
+		top: 0,
+		right: 0,
+		width: '149px',
```
Comment (sapegin) [inline src/rsg-components/Ribbon/RibbonRenderer.js]: Should work without `px`.

File: src/rsg-components/Ribbon/RibbonRenderer.js [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+		zIndex: 999,
+	},
+	link: {
+		fontFamily: fontFamily.base,
+		position: 'relative',
+		right: -37,
+		top: -22,
+		display: 'block',
+		width: 190,
+		padding: `${space[0]}px ${space[2]}px`,
```
[the remark this answers is not in the document]
Comment (glebez) [reply src/rsg-components/Ribbon/RibbonRenderer.js]: Ah, cool, thx, should have read more on jss before jumping on styles ;) will fix

File: src/rsg-components/Ribbon/RibbonRenderer.js [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+		top: -22,
+		display: 'block',
+		width: 190,
+		padding: [[space[0], space[2]]],
+		textAlign: 'center',
+		color: color.ribbonText,
+		fontSize: fontSize.base,
+		background: color.ribbonBackground,
+		textDecoration: 'none',
+		textShadow: [[0, '-1px', 0, 'rgba(0,0,0,.15)']],
```
Comment (glebez) [inline src/rsg-components/Ribbon/RibbonRenderer.js]: had to use px here, without it the code was compiling to `text-shadow: 0 -1 0 rgba(0,0,0,.15)` and browser marked it as unknown value.

**Comment by glebez** [comment]
Ok, style fixes are there.

**Comment by sapegin** [comment]
Awesome, thanks!

**Pull request merged**